django_plastic_tickets/views.py

```diff
@@ -85,11 +87,61 @@ def ticket_view(request: HttpRequest, id: int) -> HttpResponse:
     if config.user != request.user and not request.user.is_staff:
         return HttpResponseForbidden(gettext('Access denied'))
 
+    if request.method == 'POST':
+        if not request.user.is_staff:
+            return HttpResponseForbidden(gettext('Access denied'))
+        if ("close" in request.POST and
+                ticket.state == models.Ticket.TicketState.OPEN):
+            ticket.state = models.Ticket.TicketState.CLOSED
+            ticket.save()
+        elif ("reopen" in request.POST and
+              ticket.state == models.Ticket.TicketState.CLOSED):
+            ticket.state = models.Ticket.TicketState.OPEN
+            ticket.save()
+        elif "apply" in request.POST:
+            new_assignee = request.POST.get("assignee")
+            if new_assignee == "unassigned":
```

Comment:
Maybe an empty string would fit better here.

Reply:
```"unassigned"``` or an empty string can be very well distinguished from a number, which is used to set a user. So from this perspective, both are equally good. However, an empty string is not self-explanatory code, and is more often used to indicate an error or missing data.